doc/releases/changelog-dev.md

```diff
@@ -346,10 +320,10 @@ Users can specify the control wires as well as the values to control the operati
   Toffoli(wires=[2, 1, 0])
   ```
 
-* Adds caching to the `compute_matrix` and `compute_sparse_matrix` of simple non-parametric operations.
+* Added caching to the `compute_matrix` and `compute_sparse_matrix` of simple non-parametric operations.
```

Comment:
What improvement does this result in? E.g., speed, memory efficiency...

Reply:
It should result in faster computations with very little memory overhead cost.

Reply:
I don't think we benchmarked this